Analysis/Utilities/src/FnPredicates.cc

```diff
@@ -941,12 +956,9 @@ namespace ic {
     if(!pass_preselection) return false;
     double idmva = elec->GetIdIso("mvaTrigSpring15");
     if (!loose_wp) {
-      if (eta <= 0.8                    && idmva > 0.96) pass_mva = true;
-      if (eta >  0.8 && eta <= 1.479   && idmva > 0.89) pass_mva = true;
-      if (eta >  1.479                  && idmva > 0.51) pass_mva = true;
-/*      if (eta <= 0.8                    && idmva > 0.988153) pass_mva = true;
+      if (eta <= 0.8                    && idmva > 0.988153) pass_mva = true;
       if (eta >  0.8 && eta <= 1.479   && idmva > 0.967910) pass_mva = true;
-      if (eta >  1.479                  && idmva > 0.841729) pass_mva = true;*/
+      if (eta >  1.479                  && idmva > 0.841729) pass_mva = true;
```

Comment:
So we discussed updating these numbers of Skype a while back (when we were looking at using a different electron ID). The link you passed to me on the Skype message was: https://twiki.cern.ch/twiki/bin/view/CMS/MultivariateElectronIdentificationRun2#Triggering_electron_MVA_details

It looks like I've used numbers from the 25ns 80% signal eff column. Is this correct or is there more up to date numbers now?